IbFlexReader/IbFlexReader/Reader.cs

```diff
@@ -44,9 +44,12 @@ public Contracts.FlexQueryResponse GetByString(string xmlFile, Options options =
         /// <param name="queryId">ID of the FlexQuery (not the name)</param>
         /// <param name="dumpFile">if set, the data returned from IB will be dumped to file</param>
         /// <param name="retry">if GetStatement returns warning 1019 (... Please try again shortly), retry defines how many times it will be tried again</param>
+        /// <param name="retryDelay">delay between two tries in milliseconds</param>
         /// <returns></returns>
-        public async Task<FlexResult> GetByApi(string token, string queryId, string dumpFile = null, int? retry = null)
+        public async Task<FlexResult> GetByApi(string token, string queryId, string dumpFile = null, int? retry = null, int retryDelay = 3000)
         {
+            if (retryDelay <= 0) throw new ArgumentOutOfRangeException("retryDelay");
```

Comment:
please fix the codestyle https://travis-ci.org/gabbersepp/ib-flex-reader/builds/554586282?utm_source=github_status&utm_medium=notification